apps/voting/contracts/Voting.sol

```diff
@@ -335,17 +427,77 @@ contract Voting is IForwarder, AragonApp {
     * @param _voter address of the voter
     * @return VoterState of the requested voter for a certain vote
     */
-    function getVoterState(uint256 _voteId, address _voter) public view voteExists(_voteId) returns (VoterState) {
+    function getVoterState(uint256 _voteId, address _voter) external view voteExists(_voteId) returns (VoterState) {
         return votes[_voteId].voters[_voter];
     }
 
+    /**
+     * @notice Return the sliced list of delegated voters for `_delegate` and their voting power at the current block
+     * @param _delegate the address of the delegate
+     * @param _offset the number of delegated voters from the start of the list to skip
+     * @param _limit the number of delegated voters to return
+     * @return the array of delegated voters
+     * @return the array of voting power of delegated voters
+     */
+    function getDelegatedVoters(address _delegate, uint256 _offset, uint256 _limit) external view returns (address[] memory, uint256[] memory) {
+        return _getDelegatedVotersAt(_delegate, _offset, _limit, getBlockNumber64());
+    }
+
+    /**
+     * @notice Return the sliced list of delegated voters for `_delegate` and their voting power at the `_voteId` snapshot block
+     * @param _delegate the address of the delegate
+     * @param _offset the number of delegated voters from the start of the list to skip
+     * @param _limit the number of delegated voters to return
+     * @return the array of delegated voters
+     * @return the array of voting power of delegated voters
+     */
+    function getDelegatedVotersAtVote(address _delegate, uint256 _offset, uint256 _limit, uint256 _voteId) external view voteExists(_voteId) returns (address[] memory, uint256[] memory) {
+        Vote storage vote_ = votes[_voteId];
+        return _getDelegatedVotersAt(_delegate, _offset, _limit, vote_.snapshotBlock);
+    }
+
+    /**
+    * @dev Return the number of delegated voters for the `_delegate`
+    * @param _delegate address of the delegate
+    * @return the number of delegated voters
+    */
+    function getDelegatedVotersCount(address _delegate) external view returns (uint256) {
+        require(_delegate != address(0), ERROR_ZERO_ADDRESS_PASSED);
+        return delegatedVoters[_delegate].addresses.length;
+    }
+
+    /**
+     * @notice Return the delegate address assigned to the `_voter`
+     * @param _voter the address of the voter
+     */
+    function getDelegate(address _voter) external view returns (address) {
+        require(_voter != address(0), ERROR_ZERO_ADDRESS_PASSED);
+        return delegates[_voter].delegate;
+    }
+
+    /**
+     * @notice Return the list of `VoterState` for the `_voters` for the vote #`_voteId`
+     * @param _voters the list of voters
+     * @param _voteId Vote identifier
+     */
+    function getVotersStateAtVote(uint256 _voteId, address[] _voters) external view voteExists(_voteId) returns (VoterState[] memory voterStatesList) {
```

Comment:
This function is batch accessor for the `getVoterState` - it's better to name it `getVoterStateMultiple` for example

Reply:
Addressed in cd8697b8dbccbd21a4caa99e619bcf7889dd9e28.